#!/home/jhjeun86/anaconda3/envs/virtualEnv/bin/python3.6
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
